### ISSUE TITLE
Add rupee symbol to custom amount field (Other)

### ISSUE THREAD

**Issue by ekuttan** [tinkerhub-org/donation-page#69]
Rupee symbol is missing from the custom amount field in the home screen.

**Comment by monkeyscript** [comment]
@ekuttan  But that's an input field. Placeholders will disappear once the user starts to enter some value.

**Comment by monkeyscript** [comment]
We can include the symbol within placeholder. Any other idea?

**Comment by ekuttan** [comment]
Yes, @monkeyscript. We should add the prefix when user start typing.
